pkg/intermediate/aggregate.go

```diff
@@ -984,3 +993,29 @@ func isCorrelationRequired(flowType uint8, record entities.Record) bool {
 	}
 	return false
 }
+
+func fillHttpVals(incomingHttpVals, existingHttpVals string) (string, error) {
+	incomingHttpValsJson := make(map[int32]string)
```

Comment:
Forgive me if it is a dumb question. Transaction iD should be the same for the same event in different nodes?

Reply:
Yes transaction ID should be same

Reply:
I think that's a key point, and the commit message should probably mention it.
Given that the HTTP values from both Nodes should eventually be consistent, I wonder if there is real value in that "complex" aggregation logic, or if we should just use the "latest" record (like we do for some other fields) and simply override the aggregated value.

Reply:
I am not able to reproduce it in my local setup, but the thought behind this design is if multiple exporters send the httpvals with different or overlapping TxIDs, we should deduplicate it.
if we only take the latest record for aggregation, we may not get all the TxIDs in a aggregated output given the aggregation interval is bigger than the exporter interval.

Reply:
I think we won't be able to use the "latest" record in the current design.
The records from our Flow Exporter only contain partial httpVals as this field is reset after the connection is exported from the FE.
If we want to use the latest record in the FA, we might need to do the aggregation in the FE instead, which could end up increasing the packet size, transferred from FE to FA, by time?

Reply:
Thanks @yuntanghsu, that makes sense.